config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -242,3 +242,34 @@ periodics:
       - --test_args=--ginkgo.focus=\[Feature:Ingress\] --ginkgo.skip=\[Unreleased\] --minStartupPods=8
       - --timeout=300m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20181017-0323abc4c-master
+
+- interval: 6h
+  name: ci-kubernetes-e2e-kubeadm-dind-ipv6
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+    - args:
+      - --tmpfs=/docker-graph:exec
+      - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+      - --repo=github.com/kubernetes-sigs/kubeadm-dind-cluster=stable
+      - --timeout=80
+      - --clean
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --cluster=
+      - --deployment=kubeadm-dind
+      - --extract=ci/latest
+      - --kubeadm-dind-ip-mode=ipv6
+      - --provider=local
+      - --ginkgo-parallel=30
+      - --test_args=--ginkgo.focus=Networking|Services --ginkgo.skip=IPv4|DNS|Networking-Performance|Federation|functioning\sNodePort|preserve\ssource\spod|affinity\swork|affinity\sfor --minStartupPods=8
+      - --timeout=60m
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20181017-0323abc4c-master
+      # Privileged mode required to enable IPv6 and Docker-in-Docker mode
+      securityContext:
+        privileged: true
+      env:
+      - name: DOCKER_IN_DOCKER_ENABLED
+        value: "true"
```

Comment:
use `preset-dind-enabled: "true"` label, this env is deprecated

Reply:
Will switch to preset-dind-enabled.